### DISCUSSION TITLE
control of the app

### DISCUSSION THREAD

**Discussion by MjrBoobage** [JPersson77/LGTVCompanion#191] [General]
is there a way that i can manually trigger the WOL packet after the pc is on because i use the tv as a tertiary monitor and the only thing i was hoping to do with this app is to trigger the WOL by a desktop shortcut to turn on the tv are there any command line commands or something i can use?

**Comment by JPersson77** [comment]
For sure. Here is the relevant part in the command line documentation: https://github.com/JPersson77/LGTVCompanion/blob/master/Docs/Commandline.md#power-commands-lgtv-companion-and-lgtvcli

**Reply by MjrBoobage** [reply]
thank you very much ive completly solved my problem no remote anymore

**Reply by MjrBoobage** [reply]
not even the home assistant lg intergration wouldnt turn on the tv for me for some reason only the lg phone app witch is a bit how can i say it "LAGGY" so having a desktop shortcut to turn my tv on and off is fing amazing thanks again mate

**Reply by JPersson77** [reply]
You're welcome mate! :)
